qiskit/circuit/library/generalized_gates/unitary.py

```diff
@@ -123,7 +123,7 @@ def inverse(self, annotated: bool = False):
         return self.adjoint()
 
     def conjugate(self):
-        """Return the conjugate of the unitary."""
+        """Return the complex conjugate of the unitary with respect to the representation returned by ``to_matrix``."""
```

Comment:
consider linking to the method:
```suggestion
        """Return the complex conjugate of the unitary with respect to the representation returned by :meht:`~UnitaryGate.to_matrix`."""
```

Reply:
Done